includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -77,6 +79,76 @@ public function unregister_embed() {
 		remove_filter( 'embed_oembed_html', [ $this, 'filter_embed_oembed_html' ], 10 );
 	}
 
+	/**
+	 * Sanitize HTML that are not added via Gutenberg.
+	 *
+	 * @param Document $dom  Document.
+	 *
+	 * @return void
+	 */
+	public function sanitize_raw_embeds( Document $dom ) {
+
+		$nodes = $dom->xpath->query( '//iframe[ contains( @src, "youtu" ) ]' );
+
+		foreach ( $nodes as $node ) {
```

Comment:
This gives the IDE more context about the variable since it doesn't detect that it's a DOM element we're working with.

```suggestion
		/** @var Element $node */
		foreach ( $nodes as $node ) {
```

(You'd need to import `\AmpProject\Dom\Element`)